ios/FluentUI/SegmentedControl/SegmentItem.swift

```diff
@@ -18,8 +21,13 @@ public class SegmentItem: NSObject {
        }
    }
 
-    @objc public init(title: String, isUnread: Bool = false) {
+    @objc public init(title: String,
```

Comment:
would we create convenience method with deprecation on the "old" one for objc clients to migrate better?